flexmeasures/api/v3_0/users.py

```diff
@@ -45,13 +50,23 @@ class UserAPI(FlaskView):
         {
             "account": AccountIdField(data_key="account_id", load_default=None),
             "include_inactive": fields.Bool(load_default=False),
+            "page": PageField(load_default=None),
```

Comment:
Sorry, I gave you a little misleading advice (but you could also have compared with the asset API). If we only check a minimum for an integer, we should probably not introduce a whole schema, but do it like this:

```
"page": fields.Int(
    required=False, validate=validate.Range(min=1), default=1
),
"per_page": fields.Int(
    required=False, validate=validate.Range(min=1), default=10
),
```

Reply:
ok will work on that 👌